.github/pull_request_template.md

```diff
@@ -1,58 +1,38 @@
 
-<!--
-Thank you for opening a pull request!  Here are some tips on creating
-a well formatted contribution.
-
-Please give your pull request a title like "[component]: [short description]"
-
-This is the format for commit messages:
 
-"""
-[component]: [short description]
+<!--
+  - Please give your pull request a title like
 
-[A longer multiline description]
+      [component]: [short description]
 
-Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
-Signed-off-by: [Your Name] <[your email]>
-"""
+  - Please use this format for each git commit message:
 
-The Signed-off-by line is important, and it is your certification that
-your contributions satisfy the Developers Certificate or Origin.  For
-more detail, see SubmittingPatches.rst.
+      [component]: [short description]
 
-The component is the short name of a major daemon or subsystem,
-something like "mon", "osd", "mds", "rbd, "rgw", etc. For ceph-mgr modules,
-give the component as "mgr/<module name>" rather than a path into pybind.
+      [A longer multiline description]
 
-For more examples, simply use "git log" and look at some historical commits.
+      Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
+      Signed-off-by: [Your Name] <[your email]>
 
-This was just a quick overview.  More information for contributors is available here:
-https://raw.githubusercontent.com/ceph/ceph/master/SubmittingPatches.rst
+    For examples, use "git log".
 
+  - The Signed-off-by line in every git commit is important; see SubmittingPatches.rst.
 -->
+
 ## Checklist
-- [ ] References tracker ticket
-- [ ] Updates documentation if necessary
-- [ ] Includes tests for new functionality or reproducer for bug
-
----
-
-<details>
-<summary>Show available Jenkins commands</summary>
-
-- `jenkins retest this please`
-- `jenkins test classic perf`
-- `jenkins test crimson perf`
-- `jenkins test signed`
-- `jenkins test make check`
-- `jenkins test make check arm64`
-- `jenkins test submodules`
-- `jenkins test dashboard`
-- `jenkins test dashboard cephadm`
-- `jenkins test api`
-- `jenkins test docs`
-- `jenkins render docs`
-- `jenkins test ceph-volume all`
-- `jenkins test ceph-volume tox`
-
-</details>
+- Tracker (select at least one)
+  - [ ] Very recent bug; references commit where it was introduced
+  - [ ] New feature (no ticket)
+  - [ ] References tracker ticket
+  - [ ] Doc update (no ticket)
```

Comment:
Let's avoid tracker issues for #39624 as well. 

```suggestion
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature / Doc update / Maintenance (no ticket)
```

Reply:
I'd remove from the checklist anything that could be enforced automatically (like the sign-off Jenkins check).

Regarding the tracker/no-tracker discussion, since we have a "Feature" type in Redmine I'd go for tracker for everything.  

Additionally, how do we distinguish what's a feature from a bugfix? Since we're not enforcing the use of bug/feature labels, or `fix()/feat()` in commit/PR messages, only the tracker/no-tracker thing is left, but also doc PRs and other stuff can be tracker-less...

Reply:
I agree that features should have trackers too.  Sometimes features get backported and using an existing well-understood process for tracking that is good, it also gives something to reference in future bugfix tickets, etc.  But I'm not sure I would subscribe to tracker for everything.  Filing a tracker ticket for a quick fixup intended for just master is overkill IMO.  Also, an occasional contributor may not have or be willing to create a tracker account -- we don't want to discourage these drive-by or "scratching an itch" contributions with too much red tape in the PR template.